src/assets/CollectiblesController.ts

```diff
@@ -235,18 +249,40 @@ export class CollectiblesController extends BaseController<
     contractAddress: string,
     tokenId: string,
   ): Promise<CollectibleMetadata> {
-    const tokenURI = this.getCollectibleApi(contractAddress, tokenId);
-    let collectibleInformation: ApiCollectible;
+    // Attempt to fetch the data with the proxy
+    let collectibleInformation: ApiCollectible | undefined =
+      await fetchWithErrorHandling(
+        // the third arg defaults to true to use the proxy
```

Comment:
If you have to leave a comment, then I'm wondering if this means that the third argument shouldn't have a default. What do you think?

Reply:
made these positional with no default: https://github.com/MetaMask/controllers/pull/805/commits/180b2802d9edef8ff9392b88ea3a3f76cc3373da